extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxDialog.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * VisualEditor user interface WikiaInfoboxDialog class.
+ *
+ * @copyright 2011-2014 VisualEditor Team and others; see AUTHORS.txt
+ * @license The MIT License (MIT); see LICENSE.txt
+ */
+
+/**
+ * Dialog for inserting and editing Wikia Portable Infoboxes.
+ *
+ * @class
+ * @abstract
+ * @extends ve.ui.NodeDialog
+ *
+ * @constructor
+ * @param {Object} [config] Configuration options
+ */
+ve.ui.WikiaInfoboxDialog = function VeUiWikiaInfoboxDialog( config ) {
+	// Parent constructor
+	ve.ui.WikiaInfoboxDialog.super.call( this, config );
+};
+
+/* Inheritance */
+
+OO.inheritClass( ve.ui.WikiaInfoboxDialog, ve.ui.NodeDialog );
+
+/* Static Properties */
+
+ve.ui.WikiaInfoboxDialog.static.name = 'infoboxTemplate';
```

Comment:
This won't break anything but perhaps we should be more consistent here - so if name for 've.ui.WikiaInfoboxInsertDialog' is 'wikiaInfoboxInsert then name for 've.ui.WikiaInfoboxDialog' should be 'wikiaInfobox' insteaf of 'infoboxTemplate'.